example/lib/screens/no_webhook_payment_screen.dart

```diff
@@ -32,18 +50,17 @@ class _NoWebhookPaymentScreenState extends State<NoWebhookPaymentScreen> {
           Padding(
             padding: EdgeInsets.all(16),
             child: CardField(
-              controller: _editController,
-              onCardChanged: (card) {
-                setState(() {
-                  _card = card;
-                });
-              },
+              key: ValueKey('Card_ForceInitialData:$forceInitialData'),
+              dangerouslyUpdateFullCardDetails: forceInitialData,
```

Comment:
I personally wouldn't  do this on the no_webhook_payment_screen .  The dangerouslyUpdateFullCardDetails is a practice we should discourage and hide as much as possible.